lusSTR/workflows/snps.smk

```diff
@@ -14,6 +15,18 @@ def format_filename(id, refs):
         return f"{id}_snp_reference"
 
 
+def create_log(log):
+    now = datetime.now()
+    dt = now.strftime("%m%d%Y_%H_%M_%S")
+    shell("mkdir -p logs/{dt}/input/")
+    shell("cp {log} logs/{dt}/strs.log")
+    if os.path.isdir(input_name):
+        shell("cp {input_name}/* logs/{dt}/input/")
+    else:
+        shell("cp '{input_name}' logs/{dt}/input/")
+    shell("cp snp_config.yaml logs/{dt}/")
```

Comment:
Is the `log` argument to this function a path to the Snakemake logfile in `$WD/.snakemake/`?

Reply:
Yes it must be built in.